lib/AST/NameLookup.cpp

```diff
@@ -1734,12 +1734,17 @@ resolveTypeDeclsToNominal(Evaluator &evaluator,
                           SmallVectorImpl<ModuleDecl *> &modulesFound,
                           bool &anyObject,
                           llvm::SmallPtrSetImpl<TypeAliasDecl *> &typealiases) {
+  SmallPtrSet<NominalTypeDecl *, 4> knownNominalDecls;
```

Comment:
Would `SetVector<NominalTypeDecl *, TinyPtrVector<NominalTypeDecl *>, SmallPtrSet<NominalTypeDecl *, 4>>` make this simpler or more complicated?

Reply:
That's clever to simplify it. Then I can use `takeVector` to steal its vector.

Reply:
Sigh.
```
error: no type named 'size_type' in 'llvm::TinyPtrVector<swift::NominalTypeDecl *>'
```

Reply:
Sigh.